extensions/amp-story/1.0/media-pool.js

```diff
@@ -516,6 +517,11 @@ export class MediaPool {
       new SwapIntoDomTask(placeholderEl)
     ).then(
       () => {
+        if (placeholderEl.isBitrateManaged) {
```

Comment:
I think you're going to need to do the opposite of these when swapping out of dom (unmanage the pool element and manage the placeholder)